manifests/d/Dichromate/Browser/103.0.5060.114/Dichromate.Browser.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Dichromate.Browser
+PackageVersion: 103.0.5060.114
+PackageLocale: en-US
+Publisher: The Dichromate Authors
+PackageName: Dichromate Browser

```

```suggestion
PackageName: Dichromate
```
This must match control panel